src/common/policy.ts

```diff
@@ -0,0 +1,36 @@
+import { existsSync } from 'fs';
```

Comment:
Double fs import can be shortend on lines 1 and 3 :)

Reply:
Good spot